src/components/GalleryCard/index.tsx

```diff
@@ -55,10 +56,10 @@ const GalleryCard: React.SFC<GalleryCardProps> = (props) => {
         >
             3D
         </Button>,
-        (props.downloadHref !== "" || props.downloadFullField !== "") &&
-        <Dropdown key={`${props.cellID}-download`} overlay={menu} trigger={["click"]}>
+        <Tooltip key={`${props.cellID}-download`} title={hasDownload?null:NO_DOWNLOADS_TOOLTIP}>
+        <Dropdown  overlay={menu} trigger={["click"]}>
             <Button icon="download" />
-        </Dropdown>,
+        </Dropdown></Tooltip>,
```

Comment:
minor formatting nit here: indentation

Reply:
sigh, autoformatter doesn't seem to care what I do there. I'll make it cleaner.